model/offline/JsonOfflineTestParser.php

```diff
@@ -113,4 +115,13 @@ public function getTestConfig($key)
         $testConfig = $body['testConfig'];
         return (isset($testConfig[$key]) && $testConfig[$key]) ? $testConfig[$key] : null;
     }
+
+    /**
+     * @return bool
+     */
+    public function isInterrupted()
+    {
+        $body = $this->getBody();
+        return isset($body[self::KEY_IS_EXIT_TEST]) ? (bool) $body[self::KEY_IS_EXIT_TEST] : false;
```

Comment:
why `(bool)` is needed?
if `$body[self::KEY_IS_EXIT_TEST]` can be a string than it is better to use filter_var because `(bool)'false' === true`